### ISSUE TITLE
Reset form

### ISSUE THREAD

**Issue by xero88** [mozilla-services/react-jsonschema-form#953]
### Prerequisites

- [x] I have read the [documentation](https://github.com/mozilla-services/react-jsonschema-form/blob/master/README.md#readme);
- [x] In the case of a bug report, I understand that providing a [SSCCE](http://sscce.org/) example is tremendously useful to the maintainers.

### Description

What the way to reset all the form ? Like the errors displayed and the data in form

**Comment by Galibolaz** [comment]
> Simply provide a key prop to the form element, and change it whenever you wan to reset it. React will create a new fresh element for the new key.
> 
> #486

Didn't work for me. Tried resetting the formData too, but nothing seems to work.
In my case I have two different forms with the same fields. The objective here is to compare how different are people's answer to the same Form. When the first one is answered and sent to the API, the second one comes fully answered (I GET the same schema from the same endpoint and then render the Form). I gave different keys, Ids and IdPrefixes to each form, but nothing worked.
